applications/AdjointFluidApplication/custom_schemes/adjoint_bossak_scheme.h

```diff
@@ -662,6 +684,22 @@ class AdjointBossakScheme : public Scheme<TSparseSpace, TDenseSpace>
                 NUMBER_OF_NEIGHBOUR_ELEMENTS.Zero();
     }
 
+    void CalculateAdjointEnergy(ModelPart& rModelPart)
+    {
+        double total_energy = 0.0;
+        for (auto it = rModelPart.NodesBegin(); it != rModelPart.NodesEnd(); ++it) 
+        {
+            double v_x = it->FastGetSolutionStepValue(ADJOINT_FLUID_VECTOR_1_X, 0);
+            double v_y = it->FastGetSolutionStepValue(ADJOINT_FLUID_VECTOR_1_Y, 0);
+            double v_z = it->FastGetSolutionStepValue(ADJOINT_FLUID_VECTOR_1_Z, 0);
+            double p = it->FastGetSolutionStepValue(ADJOINT_FLUID_SCALAR_1, 0);
+            total_energy += (v_x*v_x+v_y*v_y+v_z*v_z+p*p);
+        }
+        ProcessInfo& rProcessInfo = rModelPart.GetProcessInfo();
+        double time = rProcessInfo[TIME];
```

Comment:
I wouldn't use `time` as a variable name bcs this is somehow specially highlighted (don't know the reason though)

Reply:
method is removed from the merge